tools/hangar/__snapshots__/test_corpus/json.w/compile/tf-aws/clients/Foo.inflight.js

```diff
@@ -1,4 +1,4 @@
-class Foo  {
+class  Foo {
```

Comment:
Since we're bothering to pretty-print stuff, should we avoid extra whitespace?

Reply:
Yes, not sure where this came from... Any idea?